tool/gravity/cli/install.go

```diff
@@ -817,13 +821,20 @@ func NewInstallerConnectStrategy(env *localenv.LocalEnvironment, config InstallC
 
 // newReconfiguratorConnectStrategy returns a new service connect strategy
 // for the agent executing the cluster reconfiguration operation.
-func newReconfiguratorConnectStrategy(env *localenv.LocalEnvironment, config InstallConfig, commandArgs cli.CommandArgs) (strategy installerclient.ConnectStrategy, err error) {
+func newReconfiguratorConnectStrategy(
+	env *localenv.LocalEnvironment,
+	config InstallConfig,
+	commandArgs cli.CommandArgs,
+) (strategy installerclient.ConnectStrategy, err error) {
+	commandArgs.FlagsToAdd = append(commandArgs.FlagsToAdd,
```

Comment:
These changes are to sync with the implementation of `NewInstallerConnectStrategy` (also below).